Configure

```diff
@@ -1642,14 +1624,12 @@ my $no_shared_warn=0;
 if (($target{shared_target} // '') eq "")
         {
         $no_shared_warn = 1
-            if (!$disabled{shared} || !$disabled{"dynamic-engine"});
+            if (!$disabled{shared});
         disable('no-shared-target', 'pic');
         }
 
 if ($disabled{"dynamic-engine"}) {
```

Comment:
same here, what is the plan for this line? Why we keeping it here?

Reply:
this can go. Thanks for noticing!